### ISSUE TITLE
give.js :: Uncaught ReferenceError: Give is not defined

### ISSUE THREAD

**Issue by lots0logs** [impress-org/givewp#3673]
## Current Behavior
My [donation form](https://antergos.com/donate) is non functional due to the following error:
```
Uncaught ReferenceError: Give is not defined
    at HTMLDocument.<anonymous> (give-2.2.4.js:11)
    at i (jquery-1.12.4.js:2)
    at Object.add [as done] (jquery-1.12.4.js:2)
    at n.fn.init.n.fn.ready (jquery-1.12.4.js:2)
    at a.fn.init.n.fn.init (jquery-1.12.4.js:2)
    at new a.fn.init (jquery-migrate.min-1.4.1.js:2)
    at n (jquery-1.12.4.js:2)
    at Object.<anonymous> (give-2.2.4.js:11)
    at t (give-2.2.4.js:1)
    at Object.<anonymous> (give-2.2.4.js:11)
```
The issue seems to only occur with Chrome. It does not happen with Firefox. I've tried debugging the issue myself for the past hour but made no progress. Any thoughts? 

## Expected Behavior
The donation form to work. without any errors in console.

## Steps to Reproduce
1. Use the link I shared above to view my donation page using Chrome.
2. See the form doesn't work.
3. Check browser console.

## Acceptance Criteria

- [ ] Something is true.
- [ ] Something happens when I do this.

## Environment
<details>
    <summary>Operating System</summary>
    <ul>
        <li>Platform: Linux</li>
        <li>Version: Arch</li>
    </ul>
</details>

<details>
    <summary>Browser</summary>
    <ul>
        <li>Name: Chrome</li>
        <li>Version: 69.0.3497.81</li>
    </ul>
</details>

<details>
    <summary>WordPress System Info</summary>
   ### WordPress Environment ###

Home URL: https://antergos.com
Site URL: https://antergos.com
WP Version: 4.9.8
WP Multisite: ✔
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: –
Language: en_US
Permalink Structure: /blog/%postname%/
Show on Front: page
Page on Front: Home (#44)
Page for Posts: Unset
Table Prefix Length: ant_
Table Prefix Length: 4
Table Prefix Status: Acceptable
Admin AJAX: Accessible
Registered Post Statuses: publish, future, draft, pending, private, trash, auto-draft, inherit, request-pending, request-confirmed, request-failed, request-completed, refunded, failed, revoked, cancelled, abandoned, processing, preapproval

### Server Environment ###

Hosting Provider: DBH: localhost, SRV: antergos.com
TLS Connection: Connection uses TLS 1.2
TLS Connection: Probably Okay
Server Info: nginx/1.14.0
PHP Version: 7.2.9
PHP Post Max Size: 40 MB
PHP Time Limit: 300
PHP Max Input Vars: 1000
PHP Max Upload Size: 40 MB
cURL Version: 7.61.0, OpenSSL/1.1.0i
SUHOSIN Installed: –
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
gzip: ✔
GD Graphics Library: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Give Configuration ###

Give Version: 2.2.4
Give Cache: Enabled
Database Updates: All DB Updates Completed.
Give Cache: Enabled
Give Cache: ✔New Donation✔Donation Receipt❌New Offline Donation❌Offline Donation Instructions✔New User Registration✔User Registration Information✔Email access
Upgraded From: 1.3.1.1
Test Mode: Disabled
Currency Code: USD
Currency Position: After
Decimal Separator: .
Thousands Separator: ,
Success Page: https://antergos.com/donation-confirmation/
Failure Page: https://antergos.com/transaction-failed/
Donation History Page: https://antergos.com/donation-history/
Give Forms Slug: /donations/
Enabled Payment Gateways: PayPal Standard
Default Payment Gateway: PayPal Standard
PayPal IPN Verification: Enabled
PayPal IPN Notifications: IPN received for #30409 ( 3K3518613R628472H ) on  at . Status VERIFIED
Donor Email Access: Enabled

### Other Active Plugins ###

Akismet Anti-Spam: by Automattic – 4.0.8
Anti-Malware Security and Brute-Force Firewall: by Eli Scheetz – 4.17.69
Bugsnag Error Monitoring: by Bugsnag Inc. – 1.3.1
Download Monitor: by Never5 – 1.9.9
Enlighter - Customizable Syntax Highlighter: by Andi Dittrich – 3.8
Force Regenerate Thumbnails: by Pedro Elsner – 2.0.6
Heartbeat Control: by Jeff Matson – 1.2.5
Join My Multisite: by Mika Epstein (Ipstenu) – 1.9.0
JSON API: by Dan Phiffer – 1.1.1
Login by Auth0: by Auth0 – 3.5.2
Mailgun: by Mailgun – 1.5.13.1
Monarch Plugin: by Elegant Themes – 1.3.16
Multisite Admin bar Switcher: by Flyn San – 1.2.6
Multi Site Plugins Add New: by Pippin Williamson – 1.1
Redis Object Cache: by Till Krüss – 1.3.8
Register IPs: by Mika Epstein, Johnny White – 1.8.0
Revisr: by Expanded Fronts, LLC – 2.0.2
Stripe Donation for WordPress: by Hal Gatewood – 1.0
Super RSS Reader: by Aakash Chakravarthy – 2.6
Theme Check: by Otto42, pross – 20160523.1
Ultimate Member: by Ultimate Member – 2.0.25
Ultimate Member - Google reCAPTCHA: by Ultimate Member – 1.0.0
User Role Editor: by Vladimir Garagulya – 4.45
Wordfence Security: by Wordfence – 7.1.11
WordPress Zero Spam: by Ben Marshall – 3.0.6
WP Rocket: by WP Media – 3.1.3.2
Yoast SEO: by Team Yoast – 8.2

### Inactive Plugins ###

Frontend Publishing Pro: by Hassan Akhtar – 2.82
Give - Form Field Manager: by WordImpress – 1.1
Give - PDF Receipts: by WordImpress – 2.0.6
Give - Stripe Gateway: by WordImpress – 1.5
MG POST Contributors: by Mahesh Waghmare – 1.3.
Redirection: by John Godley – 3.4.1
Relevanssi: by Mikko Saari – 4.0.11
Simple Category Icons: by Bas Schuiling – 1.12
Table of Contents Plus: by Michael Tran – 1601
Top 10: by Ajay D'Souza – 2.5.6
Transposh Translation Filter: by Team Transposh – 0.9.9.1
WordPress Importer: by wordpressdotorg – 0.6.4
WP REST API: by WP REST API Team – 2.0-beta15

### Theme ###

Name: Divi-Antergos
Version: 
Author URL: http://www.elegantthemes.com
Child Theme: Yes
Parent Theme Name: Divi
Parent Theme Version: 3.14
Parent Theme Author URL: http://www.elegantthemes.com

</details>

**Comment by Benunc** [comment]
I just attempted the form there, and was taken away to paypal. You should see a pending donation in the DOnations table on the back end of the site. This is most likely Chrome aggressively caching things on the page. 

This is not the best forum for these types of requests, so I am closing this. Our support team is happy to answer your questions via support: https://givewp.com/support

**Comment by lots0logs** [comment]
@Benunc I think this issue boils down to give.js being enqueued on all frontend pages instead of only pages that contain a donation form. Thoughts?

**Comment by Perchpole** [comment]
Did you get this resolved? I have the same problem.

**Comment by lots0logs** [comment]
No I'm afraid not. We shut down our project so are no longer using Give.

**Comment by ravinderk** [comment]
@lots0logs When click on the `Donate` button after revealing form. I find that any other plugin is conflicting with `Give`. did you test Give after disabled all plugin?

In my case: I found that when I clicked on the `Donate` button then I found that one string added after button which seems social media stat. 

![image](https://user-images.githubusercontent.com/1784821/67454666-af431c80-f648-11e9-9297-532da74474b0.png)
